domains/authorization/Authorization.API/API.IntegrationTests/API.IntegrationTests/Setup/TestWebApplicationFactory.cs

```diff
@@ -119,15 +118,20 @@ private string GenerateToken(string sub, string name, string orgIds, string subT
 
         var signingCredentials = new SigningCredentials(new X509SecurityKey(cert), SecurityAlgorithms.RsaSha256);
         var tokenHandler = new JwtSecurityTokenHandler();
-        var identity = new ClaimsIdentity(new List<Claim>
+        var claims = new List<Claim>
         {
             new("sub", sub),
             new("name", name),
             new("org_ids", orgIds),
             new("sub_type", subType),
             new("org_cvr", orgCvr),
-            new("org_name", orgName)
-        });
+            new("org_name", orgName),
+        };
+        if (termsAccepted)
+        {
+            claims.Add(new("tos_accepted", true.ToString()));
```

Comment:
Will this claim never på false? just not existing if not accepted?

Reply:
Correct, terms is accepted if this claim is present and has value true. Other values or non-existing claim means terms has not been accepted.